browser/css/jsdialogs.css

```diff
@@ -44,6 +44,7 @@
 img.ui-drawing-area {
 	width: -webkit-fill-available;
 	width: -moz-available;
+	padding: 5px;
```

Comment:
please remove this, we should avoid adding margins/padding to "image" widgets as they sit in scrolledwindows and we do some calculations based on it's size, this can break something